tests/infrared/16.1/infrared-openstack.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
```

Comment:
This is a fundamental change, I has thinking of a better name, but since I was using this with 16.1, it became 16.1.

Fundamental change, since it also deploys gnocchi (on ceph). With 5 seconds interval, (in ceilometer, same as collectd), it puts a high load on your ceph, so this won't deploy any production ready cloud, but it is well suited for smaller and mid-size tests.

Reply:
I still feel like this configuration is way too much copy/pasta from another script. I'd like to see this setup in a way that we could either just pass other environment files, or something, so that we're not copying over so much of the 16/ script to this new 16.1/ script.

I know the environment file is different and has a different setup environment, but it's still all the same bash functions etc. I really want to avoid duplicating so much of the original script.